modules/custom-status/custom-status.php

```diff
@@ -1602,7 +1602,7 @@ function fix_get_sample_permalink_html( $return, $post_id, $new_title, $new_slug
 	 *
 	 * @since 0.8
 	 */
-	private function get_preview_link( $post ) {
+	static function get_preview_link( $post ) {
```

Comment:
I know that the visibility declaration of methods in this file is not uniform, but we, imho, should follow the best practices declared in the [WordPress coding standards project](https://github.com/WordPress-Coding-Standards/WordPress-Coding-Standards/blob/805346356689075d9f40b57cbedd05bc813eb5ba/WordPress-Extra/ruleset.xml#L93), specifically the WordPress-Extra ruleset, which sets up best practices beyond the core WordPress coding standards.

And thus, the `public` visibility declaration should be used here. It would also help to highlight the change, as this method becomes public by removing the `private` visibility.

Reply:
Thank you for the review!

I checked the [WordPress Coding Standards](https://github.com/WordPress-Coding-Standards/WordPress-Coding-Standards/issues/1101)  and I can see that the `Methods: Verify that all methods have visibility declared`. Does this mean that I should use

`public function get_preview_link( $post )`

instead of

`function get_preview_link( $post )`

am I right? I just want to make sure that I follow the Coding Standards and write this the right way.

Reply:
Right. `public static function get_preview_link( $post )` to be exact.